tools/romimg/src/romimg.c

```diff
@@ -6,11 +6,16 @@
 #include <stdio.h>
 #include <stdlib.h>
 #include <string.h>
+#include <unistd.h>
+#include <limits.h>
 
 #include "platform.h"
 #include "romimg.h"
 #include "SonyRX.h"
 
+#define BUFCHK(X) (X[0] == '\0') ? "" : X
```

Comment:
Do you think this is really needed? 
I think it makes no sense at all, the behavior should be the same, no matter if the first character is `\0`

Reply:
yeah, it remained from older approach. will deal with it now